gpjax/abstractions.py

```diff
@@ -2,13 +2,17 @@
 
 import jax
 import jax.numpy as jnp
+import jax.random as jr
 import optax
+from chex import PRNGKey
```

Comment:
Is this intentional? Could we perhaps alias the import as `PRNGKeyType`

Reply:
Good spot! I did not give this much thought, I followed what was done in distrax (see e.g., https://github.com/deepmind/distrax/blob/615605e8701eb55f75854b9b6e13913a9d6d7716/distrax/_src/distributions/distribution.py). Your suggestion of `PRNGKeyType` makes more sense!